docs/driver-controls-card.md

```
@@ -176,14 +177,48 @@ Let go early and nothing comes out.
 
 ---
 
+## Turbo ⚡
+
+**Flightstick 9 · Xbox right bumper**
+
+Hold it for extra pushing power. It is there for **getting over the bump with a full
+hopper**, and for shoving matches. Hold it, drive into the thing, let go once you're over.
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)